pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -77,15 +89,29 @@ func (enqueue *enqueueAction) Execute(ssn *framework.Session) {
 	emptyRes := api.EmptyResource()
 	nodesIdleRes := api.EmptyResource()
 	for _, node := range ssn.Nodes {
-		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2).Sub(node.Used))
+		nodesIdleRes.Add(node.Allocatable.Clone().Multi(1.2))
```

Comment:
1. the Clone of Allocatable is unnecessary to me.
1. multi 1.2 out of for loop.

Reply:
method 'Multi' will change the object itself, Clone required if we put the logic inside of for loop